app/views/posts/index.html.erb

```diff
@@ -24,7 +24,8 @@
         </p>
     </div>
     <div class='comment'>
-    <% @comment.each do |comments|%>
+    <% @comment.each_with_index do |comments, idx|%>
+        <% break if idx == 5 %>
```

Comment:
- [ ] The same happens here, please consider using the method you already created:

https://github.com/francksefu/WebApp/blob/a63e0e7a861dd92dac0a0fb3f3d3832d675f3d65/app/models/post.rb#L10-L13